price_feeds_poller.py

```diff
@@ -115,11 +115,121 @@ def handle_requestUpdate(
         print(f" ==== Previous query id : {feed_latest_update_query_id}\n")
         return [ feed_latest_update_query_id, tx.hex(), total_fee ]
 
+def reload_config(config_file_path, network_name):
+    config = load_dfe_config(config_file_path)
+    network_config = get_network_config(config, network_name)
+    if network_config['version'] != '2.0':
+      print(f"Fatal: Network '{network_name}' not prepared for 2.0")
+      exit(1)
+    return config, network_config.get('address', config['contracts']['2.0']['address'])
+
+def reload_pfs(feeds, config, network_name):
+    
+  captionMaxLength = 0
+  ids = []
+  pfs = []    
+
+  supports = feeds.functions.supportedFeeds().call()
+  for index in range(len(supports[0])):
+    caption = supports[1][index]
+    pf_id = supports[0][index].hex()
+    rad_hash = supports[2][index].hex()
```

Comment:
I think we can use `zip` here.
```suggestion python
  for (pf_id_bytes, caption_bytes, rad_hash_bytes) in zip(*supports):
    caption = caption_bytes.decode()
    pf_id = pf_id_bytes.hex()
    rad_hash = rad_hash_bytes.hex()
```